aries-site/src/themes/aries.js

```diff
@@ -4,16 +4,19 @@ import { deepMerge, normalizeColor } from 'grommet/utils';
 export const aries = deepMerge(hpe, {
   defaultMode: 'dark',
   formField: {
-    border: {
-      error: {
-        color: 'text-weak',
-      },
-    },
-    error: {
+    disabled: {
       background: {
-        color: { light: '#FF40404D', dark: '#FF404066' },
+        color: undefined,
       },
     },
+    extend: ({ disabledStyle, theme }) => {
+      return `
+        ${disabledStyle &&
+          `div { border-color: ${
```

Comment:
Will this still be needed even after the grommet PR? feels a little abnormal to have to use this div

Reply:
It will not. I will remove. I didn't like it either.